prime-dotnet-webapi/Services/SoapService.cs

```diff
@@ -22,9 +21,40 @@ public class SoapServiceOperationTuner : IServiceOperationTuner
 
         public void Tune(HttpContext httpContext, object serviceInstance, SoapCore.ServiceModel.OperationDescription operation)
         {
+            // This must be the header used by proxies that first receive the client certificate
+            // (and pass on via HTTP Header)
+            const string CLIENT_CERT_HEADER = "X-SSL-CERT";
```

Comment:
There are already some constants for this service defined on lines 19/20, do you think this should go there?
Also, I tend to avoid using screaming snake case, preferring to have consistent PascalCase instead.